.github/workflows/build_wheels.yml

```diff
@@ -12,6 +12,7 @@ jobs:
     name: Build wheels on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
```

Comment:
I added this for easier debugging. That one build does not finish because of another. E.g. the Windows build does not finish because of failing MacOS. The whole building will still fail at the end, and no upload will happen.